.github/workflows/regenerate-tools.yml

```diff
@@ -13,15 +13,22 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
       name: Regenerate tools.json
       runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          node-version: [16.x]
```

Comment:
`strategy` is recommended in cases when you use more than one version. Like we do here for example: https://github.com/asyncapi/.github/blob/master/.github/workflows/if-nodejs-pr-testing.yml#L17. So we use strategy to make sure workflow runs 3 times on 3 different node versions.

in case of this workflow we need to run it only one, on specific node version, so we not just use it in this one new step: actions/setup-node@v3

Reply:
Yepp, done. Should we merge this PR in another branch for testing workflow?